app/models/library_entry.rb

```diff
@@ -33,6 +33,14 @@ class LibraryEntry < ActiveRecord::Base
   VALID_STATUSES = ["Currently Watching", "Plan to Watch", "Completed", "On Hold", "Dropped"]
   validates :status, inclusion: {in: VALID_STATUSES}
 
+  SNAKE_PARAMETERS = {
```

Comment:
I don't like this constant name because it don't relate to the hash content. maybe something like `SNAKE_STATUSES` or `STATUSES_HASH`?